Makefile

```diff
@@ -86,13 +86,15 @@ venv:
 # We also install cyclonedx-go to generate SBOM for Go, compile the Go modules,
 # install SLSA verifier binary, and download mvnw.
 .PHONY: setup
+.PHONY: souffle
```

Comment:
Can you please move this down closer to the target?

Reply:
Applied in https://github.com/oracle-samples/macaron/pull/46/commits/1efa2349a079341ec72316b8c00d677e0a91e615